README.md

```diff
@@ -3,8 +3,12 @@
 This repository contains the capstone project for a pod of interns (step1-2020)
 in Google's Student Training in Engineering program.
 
+## Set Up
+
 This project requires JDK 11.
 
+This project also requires an [API Key](https://developers.google.com/custom-search/v1/introduction) and three total [Google Custom Search Engine IDs](https://cse.google.com/cse/all) added to the `SearchServlet.java` file. One per restricted site client: [StackOverflow](https://stackoverflow.com/), [W3Schools](https://www.w3schools.com/), [GeeksForGeeks](https://www.geeksforgeeks.org/).
```

Comment:
Nit: ...added to the `SearchServlet.java` file, one per restricted... (so that we have complete sentences)

This line is 389 characters long, can you limit each line to 80 characters?